package.json

```diff
@@ -60,6 +60,7 @@
     "request": "^2.67.0",
     "semver": "5.1.0",
     "sha.js": "^2.4.4",
+    "shelljs": "0.6.0",
```

Comment:
Oops, no, it was for a part I split into a second branch and forgot to remove. I’ll take it out. Thanks ^_^